core/new-gui/protractor.conf.js

```diff
@@ -0,0 +1,28 @@
+// Protractor configuration file, see link for more information
+// https://github.com/angular/protractor/blob/master/lib/config.ts
+
+const { SpecReporter } = require('jasmine-spec-reporter');
+
+exports.config = {
+  allScriptsTimeout: 11000,
+  specs: [
+    './e2e/**/*.e2e-spec.ts'
+  ],
+  capabilities: {
+    'browserName': 'chrome'
+  },
+  directConnect: true,
```

Comment:
The github page lists that we should give a driver path. But it's not here. Have we tested this config? Is there a source where a config in this form is listed?

Reply:
This file is automatically generated by Angular CLI when creating a new project, you ignore can all these config files.